main.py

```diff
@@ -30,7 +31,7 @@
         db.create_all()
```

Comment:
I don't see a function definition for this but I do see `create`.

Reply:
I also don't see a function declaration for `init_app`. Maybe I'm missing something though.

Reply:
This is flask-sqlalchemy initialization stuff. [See here](https://flask-sqlalchemy.palletsprojects.com/en/3.0.x/quickstart/)

So `db` is an instance of the `SQLAlchemy` class (see in common.py I call `db = SQLAlchemy()`, but `Agent` is a class inheriting from `db.Model` and it represents an agent in the database. The way I wrote the Agent class is based a bit [on this](https://github.com/cookiecutter-flask/cookiecutter-flask/blob/master/%7B%7Bcookiecutter.app_name%7D%7D/%7B%7Bcookiecutter.app_name%7D%7D/database.py#L15
)

Reply:
Ohhh sorry, I missed that.